src/components/SearchInput/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ClassNames from 'classnames';
+
+import styles from '@/components/SearchInput/SearchInput.css';
+
+import IconSearch from '@/assets/images/icons/web/search-simple.svg';
+import IconClose from '@/assets/images/icons/web/close.svg';
+
+type Size = 'large' | 'medium';
+type State = '' | 'valid' | 'invalid';
+
+interface Props {
+  clearInputClick: () => void;
+  description?: string;
+  disabled?: boolean;
+  label?: string;
+  message?: string;
+  onChange: () => void;
+  placeholder?: string;
+  state?: State;
+  size: Size;
+  value?: string;
+  variablesClassName?: string;
+}
+
+const SearchInput: React.FC<Props> = props => {
+  const {
+    clearInputClick,
+    description,
+    disabled,
+    label,
+    onChange,
+    message,
+    placeholder,
+    size,
+    state,
+    value,
+    variablesClassName,
+    ...searchInputProps
+  } = props;
+  const sizeClass = `search--${size}`;
+  const statusClass = state ? `search--${state}` : '';
+  const sizeInputClass = `search-input--${size}`;
+  const messageValidateClass = state ? `search-input--message-${state}` : '';
+
+  return (
+    <div className={ClassNames(variablesClassName)}>
+      {label ? <label className={ClassNames(styles['search-label'])}>{label}</label> : ''}
+      {description ? (
+        <span className={ClassNames(styles['search-description'])}>{description}</span>
+      ) : (
+        ''
+      )}
+      <div className={ClassNames(styles.search, styles[sizeClass], styles[statusClass])}>
+        <button className={ClassNames(styles['search-icon-button'])}>
```

Comment:
I'm not sure if this should be an interactive element with the keyboard, maybe this would be enough

```
<IconSearch height='15' width='15' title='Search Icon' />
```

I'd just suggest adding an `onClick` event to the `IconSearch` in order to focus the input if a user clicks on the icon